client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -214,12 +246,19 @@ const BrewRenderer = (props)=>{
 					onKeyDown={handleControlKeys}
 					tabIndex={-1}
 					style={{ height: state.height }}>
+
+					<ErrorBar errors={props.errors} />
+					<div className='popups'>
+						<RenderWarnings />
+						<NotificationPopup />
+					</div>
```

Comment:
Here it is. Errorbar and Popups have been added here but they are already present above, outside the iFrame.

Reply:
Probably a conflict badly resolved, we will delete this.